extensions/positron-r/src/r-installation.ts

```diff
@@ -336,12 +336,51 @@ function getExcludedInstallations(): string[] {
 }
 
 /**
- * Checks if the given binary path is excluded via settings.
+ * Gets the list of R installations to override the installations we make available.
+ * The override setting take precedence over the excluded installations, the custom binaries
+ * and the custom root folders settings.
+ * Converts aliased paths to absolute paths. Relative paths are ignored.
+ * @returns List of installation paths to exclusively include.
+ */
+export function getInterpreterOverridePaths(): string[] {
+	const config = vscode.workspace.getConfiguration('positron.r');
+	const interpretersOverride = config.get<string[]>('interpreters.override') ?? [];
+	if (interpretersOverride.length > 0) {
```

Comment:
Very weakly held opinion: I think I'd check for length zero and get out early. I'm just a big fan of early exits.